FrescoImageLoader/src/main/java/com/github/piasy/biv/loader/fresco/ImageDownloadSubscriber.java

```diff
@@ -77,8 +77,14 @@ protected void onNewResultImpl(DataSource<CloseableReference<PooledByteBuffer>>
         } catch (IOException e) {
             onFail(e);
         } finally {
-            IOUtils.closeQuietly(inputStream);
-            IOUtils.closeQuietly(outputStream);
+            try {
```

Comment:
Why not keep `closeQuietly` in the copied `IOUtils` class?

Reply:
Means our own try catch method with closeQuietly name??